### PR TITLE
add docs for autoupdater

### PR DESCRIPTION
**Pull request by tomkennedy513**

### CODE REVIEW

**Review by mgibson1121 — APPROVED** [review]
Given that these docs are copied and pasted over to several locations, going to leave this as a general comment.  I wonder if we should state that the descriptor is the thing updater is monitoring.  This will make it more clear how frequently they can expect updates in their store, builder, and stack resources and why they need to provide tanzunet creds.

**Review by dumez-k — APPROVED** [review]
Looks great

File: installing.html.md.erb [hunk 1]
```diff
@@ -179,6 +199,8 @@ Where:
 
 ### <a id='import'></a> Import Tanzu Build Service Dependencies
 
+<p class="note"><strong>Note:</strong> Tanzu Build Service 1.2 ships with a automatic dependency updater. If you have enabled this feature durign install, you can skip this step. To check if you have a TanzuNetDependencyUpdater in your cluster, run: `kubectl get TanzuNetDependencyUpdaters -A` </p>
```
[the remark this answers is not in the document]
Comment (tylerphelan) [reply installing.html.md.erb:202]: `durign`

[the remark this answers is not in the document]
Comment (matthewmcnew) [reply installing.html.md.erb]: 🥳

**Review by matthewmcnew — APPROVED** [review]

File: installing.html.md.erb [hunk 2]
```diff
@@ -451,7 +473,25 @@ imgpkg copy -b "registry.pivotal.io/build-service/bundle:<TBS-VERSION>" --to-rep
...
+	-v docker_password="<REGISTRY-PASSWORD>" \
+	-v tanzunet_username="<TANZUNET-USERNAME>" \
+	-v tanzunet_password="<TANZUNET-PASSWORD>" \
+	| kbld -f /tmp/bundle/.imgpkg/images.yml -f- \
+	| kapp deploy -a tanzu-build-service -f- -y
+```
+
+You can check the status of the DependencyUpdater by running `kubectl -n build-service get TanzuNetDependencyUpdater dependency-updater -o yaml` 
+
+Alternatively, if you prefer to manage dependencies yourself, simply leave the credentials out of the install.
```
Comment (tylerphelan) [inline installing.html.md.erb:494]: maybe say "simply leave the Tanzu Net credentials out of install"

**Comment by tylerphelan** [comment]
maybe add 
```
<p class="note"><strong>Note:</strong> The DependencyUpdater cannot be used in air-gapped environments. Do not include Tanzu Net credentials for air-gapped installations.</p>
```

 to air-gapped section

**Pull request merged**